moped-database/views/combined_project_funding_view.sql

```diff
@@ -26,15 +26,15 @@ LEFT JOIN moped_fund_programs ON moped_proj_funding.funding_program_id = moped_f
 WHERE moped_proj_funding.is_deleted = FALSE
 UNION ALL
 SELECT
-    'ecapris_'::text || ecapris_subproject_funding.id AS id,
+    (('ecapris_'::text || ecapris_subproject_funding.id) || '_moped_'::text) || moped_project.project_id AS id,
     ecapris_subproject_funding.id AS original_id,
     ecapris_subproject_funding.created_at,
     ecapris_subproject_funding.updated_at,
-    NULL::integer AS project_id,
+    moped_project.project_id,
     ecapris_subproject_funding.fdu,
     ecapris_subproject_funding.unit_long_name,
     ecapris_subproject_funding.app AS amount,
-    'Synced from eCAPRIS'::text AS description,
+    NULL::text AS description,
```

Comment:
Removed based on user and dev feedback. We can conditionally show UI icons, labels, or whatever is needed using `is_synced_from_ecapris` instead of repurposing this description column.